### PR TITLE
feat: `/students` marketing page

### LINKED ISSUE #222: Feature: Students landing page
### Type of feature

🍕 Feature

### Current behavior

N/A

### Suggested solution

We need to build a landing page for Students, for the upcoming Github Students collaboration.

Due date is March 26th so we need to move fast.

### Tasks

- [x] Copy (@BekahHW @isabensusan @chhristopher)
- [ ] Images (@isabensusan)
- [x] Setup page (@zeucapua)

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by zeucapua**
## Description

Implements `/students` page with data from Sanity CMS.

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Closes #222 

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: components/sections/navigation/DesktopNavigation.tsx [hunk 1]
```diff
@@ -25,11 +25,16 @@ const useCases = [
...
     description: 'Grow and celebrate your open source community.',
   },
-  {
-    link: '/contributors',
-    title: 'Contributors',
-    description: 'Start the path to your next contribution.',
+  { 
+    link: "/contributors",
+    title: "Contributors", 
+    description: "Start the path to your next contribution."
```
Comment (jpmcb) [inline components/sections/navigation/DesktopNavigation.tsx:31]: Related: https://github.com/open-sauced/landing-page/issues/227

**Review by jpmcb — APPROVED** [review]
Looks good to me! 🎸

**Review by nickytonline — COMMENTED** [review]
The changes look good, but for some reason the image coming from Sanity's CDN aren't loading. Not sure if this is just a deployment preview issue though. cc: @BekahHW 

CleanShot 2024-03-19 at 12 56 40

**Comment by isabensusan** [comment]
> The changes look good, but for some reason the image coming from Sanity's CDN aren't loading. Not sure if this is just a deployment preview issue though. cc: @BekahHW
> 
> CleanShot 2024-03-19 at 12 56 40

It's loading fine! this is just a placeholder for the images :)

also, LGTM 💯

**Pull request merged**